From 030da20f1c0713867100dcbec1cebd66e57db2a0 Mon Sep 17 00:00:00 2001
From: Arun Murthy <acmurthy@apache.org>
Date: Mon, 24 Dec 2012 00:14:59 +0000
Subject: [PATCH 0844/1357] HDFS-4326. bump up Tomcat version for HttpFS to 6.0.36. Contributed by Alejandro Abdelnur.

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/trunk@1425560 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 5d72eb6005e9f4792e4c1064e952f41ae2d61f96)

Conflicts:
	hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml
---
 hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml b/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml
index caeeafa..f3cefd8 100644
--- a/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml
+++ b/hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml
@@ -31,7 +31,7 @@
   <description>Apache Hadoop HttpFS</description>
 
   <properties>
-    <tomcat.version>6.0.32</tomcat.version>
+    <tomcat.version>6.0.36</tomcat.version>
     <httpfs.source.repository>REPO NOT AVAIL</httpfs.source.repository>
     <httpfs.source.repository>REPO NOT AVAIL</httpfs.source.repository>
     <httpfs.source.revision>REVISION NOT AVAIL</httpfs.source.revision>
@@ -521,7 +521,7 @@
                     <mkdir dir="downloads"/>
                     <get
                         src="http://archive.cloudera.com/tarballs/apache-tomcat-${tomcat.version}.tar.gz"
-                        dest="downloads/tomcat.tar.gz" verbose="true" skipexisting="true"/>
+                        dest="downloads/apache-tomcat-${tomcat.version}.tar.gz" verbose="true" skipexisting="true"/>
                     <delete dir="${project.build.directory}/tomcat.exp"/>
                     <mkdir dir="${project.build.directory}/tomcat.exp"/>
 
@@ -535,7 +535,7 @@
                       BUILD_DIR=`cygpath --unix '${project.build.directory}'`
                       fi
                       cd $BUILD_DIR/tomcat.exp
-                      tar xzf ${basedir}/downloads/tomcat.tar.gz
+                      tar xzf ${basedir}/downloads/apache-tomcat-${tomcat.version}.tar.gz
                     </echo>
                     <exec executable="sh" dir="${project.build.directory}" failonerror="true">
                       <arg line="./tomcat-untar.sh"/>
-- 
1.7.0.4

